src/main.ts

```diff
@@ -89,6 +90,13 @@ export class MatterAdapter extends utils.Adapter {
     #nodeReSyncInProgress = new Set<string>();
     #closing = false;
     #version: string = '0.0.0';
+    #objectProcessQueue = new Array<{
```

Comment:
Any reason for this uncommon syntax?

Reply:
for me new Array is common in the meantime and easy because in sync with Set/Map and such comparable types